db/migrate/20240611150632_add_booking_prices.rb

```diff
@@ -0,0 +1,15 @@
+class AddBookingPrices < ActiveRecord::Migration[7.1]
+  def change
+    create_table :booking_prices do |t|
+      t.decimal :price, scale: 3, precision: 12, null: false
```

Comment:
`BookingPrice.price` isn't the best way to express this. I would consider `value or amount` as alternative, but for this tech test I'm going to keep `price` for consistency with the other price fields used on other models.